src/lib/templateHelpers.ts

```diff
@@ -103,5 +98,26 @@ export const formatForTsDoc = (str: string): string => {
   return collapsedLeadingWhitespace;
 };
 
-import { addNamespace } from "./commonTemplateHelper";
-export { addNamespace };
+/**
+ * Checks if a path parameter is one of the set that are configurable at the client level
+ *
+ * @param property - The string name of the parameter to check
+ *
+ * @returns true if the parameter is a common parameter
+ */
+export const isCommonPathParameter = (property: string): boolean =>
+  property
+    ? commonParameterPositions.pathParameters.includes(property.toString())
```

Comment:
Why do we need to call toString() on string?

Reply:
Don't know, didn't change this.